esmvalcore/cmor/_fixes/cmip5/gfdl_cm2p1.py

```diff
@@ -56,6 +58,50 @@ def fix_data(self, cube):
         return cube
 
 
+class Sit(Fix):
+    """Fixes for sit"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes bad bounds
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        time = cube.coord('time')
+        if self._fix_required(time):
+            new_bounds = np.empty(time.bounds.shape, time.bounds.dtype)
+            for x, point in enumerate(time):
```

Comment:
```suggestion
            for i, point in enumerate(time):
```
would be more in line with the conventions